rootfs/api/migrations/0001_initial.py

```diff
@@ -20,7 +20,7 @@ class Migration(migrations.Migration):
                 ('uuid', api.fields.UuidField(auto_created=True, primary_key=True, serialize=False, editable=False, max_length=32, unique=True, verbose_name='UUID')),
                 ('created', models.DateTimeField(auto_now_add=True)),
                 ('updated', models.DateTimeField(auto_now=True)),
-                ('id', models.SlugField(max_length=24, unique=True, null=True, validators=[api.models.validate_id_is_docker_compatible, api.models.validate_reserved_names])),
+                ('id', models.SlugField(max_length=24, unique=True, null=True, validators=[api.models.validate_app_id, api.models.validate_reserved_names])),
```

Comment:
Actually it is fine to update the existing migrations. We actually need to do this anyways because we hit an error when attempting to create a new migration:

```
  File "/home/bacongobbler/go/src/github.com/deis/controller/rootfs/api/migrations/0002_auto_20151215_0352.py", line 17, in Migration
    field=models.SlugField(max_length=24, unique=True, null=True, validators=[api.models.validate_id_is_docker_compatible, api.models.validate_reserved_names]),
AttributeError: module 'api.models' has no attribute 'validate_id_is_docker_compatible'
```

So the code as-is should be fine for existing deployments. IIRC we have done this in previous PRs and this was OK since django doesn't actually store this information in the migration database.